mesonbuild/minstall.py

```diff
@@ -431,10 +431,12 @@ def do_copyfile(self, from_file: str, to_file: str,
         append_to_log(self.lf, to_file)
         return True
 
-    def do_symlink(self, target: str, link: str, full_dst_dir: str, allow_missing: bool) -> bool:
+    def do_symlink(self, target: str, link: str, destdir: str, full_dst_dir: str, allow_missing: bool) -> bool:
```

Comment:
Actually I wonder. We use both target and link, but we don't "use" full_destdir, we just need it to define the abs_target. I am not sure whether here or at the call site is maybe a better place to calculate this?
(I genuinely am not sure.)

Reply:
I was thinking about that, but `full_dst_dir` is still used by parent to create the directory, 

We could definitely do that under symlink, but we'd still need to bring the `DirMaker` with us